sources/packages/backend/apps/api/src/route-controllers/student/student.institutions.controller.ts

```diff
@@ -36,11 +38,14 @@ export class StudentInstitutionsController extends BaseController {
 
   /**
    * Search students based on the search criteria.
+   * Return a 200 HTTP status instead of 201 to indicate that the operation
+   * was completed with success but no resource was created.
    * TODO add decorator to restrict to BC Public institutions.
    * @param searchCriteria criteria to be used in the search.
    * @returns searched student details.
    */
   @Post("search")
+  @HttpCode(HttpStatus.OK)
```

Comment:
If you are in the mood, the same can be done on the AEST search if no side effects.